_posts/2021-07-17-signing-disruption.markdown

```diff
@@ -0,0 +1,14 @@
+---
+title: Repository Signing Disruption
+layout: post
+---
+
+We are aware of an ongoing infrastructure issue that is affecting
+repository syncronization and package signing.  We are working to
+resolve these issues, but first must resolve another unrelated
```

Comment:
Perhaps make it clear whether it is one issue or not, in first sentence you say "issue" but here it is "issues"